Commands/Commands/uservoice.js

```diff
@@ -129,6 +129,83 @@ commands.submit = {
   }
 }
 
+commands.status = {
+  adminOnly: true,
+  modOnly: false,
+  fn: function (bot, msg, suffix, uvClient, cBack) {
+    msg.channel.sendTyping()
+    getMail(uvClient, msg.author.id).then(email => {
+      uvClient.v1.loginAs(email).then(c => {
+        var grabidregex = /(none|x|review|re|r|under review|ur|u|planned|plan|pl|p|started|start|st|s|completed|complete|co|c|declined|decline|de|d|1|2|3|4|5|6)+(.{0,})/i;
+        let s = suffix.split(' ');
+        let idt = s[0];
+        s.shift();
+        let statncontent = s.join(' ');
+        let grabstatusidfromhere = statncontent.match(grabidregex);
+        let statusname = grabstatusidfromhere[2];
+        let statusid = null;
+        if (statusname.match(/(review|re|r|under review|ur|u|2)/)) { statusid = '1707873'; statusname = 'Under Review'; }
+        else if (statusname.match(/(planned|plan|pl|p|3)/)) { statusid = '1707876'; statusname = 'Planned'; }
+        else if (statusname.match(/(started|start|st|s|4)/)) { statusid = '1707879'; statusname = 'Started'; }
+        else if (statusname.match(/(completed|complete|co|c|5)/)) { statusid = '1707882'; statusname = 'Completed'; }
+        else if (statusname.match(/(declined|decline|de|d|6)/)) { statusid = '1707885'; statusname = 'Declined'; }
+        else if (statusname.match(/(none|x|1)/)) { statusid = null; statusname = 'None'; }
+        let parts = idt.match(UVRegex)
+        let id
+        if (parts === null) {
```

Comment:
make this `id = parts === null ? idt : parts[2]` or `id = (parts && parts[2]) || idt`